modules/cluster-logging-forwarding-about.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging.adoc
+
+[id="cluster-logging-forwarding-about_{context}"]
+= About log forwarding
+
+By default, {product-title} cluster logging sends logs to the default internal Elasticsearch log store, defined in the Cluster Logging Custom Resource. If you want to forward logs to other log aggregators, you can use the log forwarding features to send logs to specific endpoints within or outside your cluster.
```

Comment:
I believe we are now using lowercase here ("Cluster Logging custom resource"), per the [note](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#object-references) that was updated in our guidelines:
> You can use "object", "resource", "custom resource", "spec", etc. as appropriate after the object reference. This is especially recommended if it helps with clarity or readability.

Reply:
@bobfuru 
Agreed. I left it upper-case as it appears that way throughout the logging docs. I should do a universal change.